v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.monster.etl.v2f
+
+import better.files._
+import com.spotify.scio.testing._
+import io.circe.Json
+import org.scalatest.BeforeAndAfterAll
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.scalatest.Matchers
+
+class V2FIntegrationSpec extends PipelineSpec with Matchers with BeforeAndAfterAll {
+
+  behavior of "V2F ETL Pipeline"
+
+  private val truthDir = "v2f" / "src" / "it" / "test-files" / "outputs"
+  private val compareDir = "v2f" / "src" / "it" / "test-files" / "outputs-to-compare"
+  private val inputDirString = "v2f/src/it/test-files/inputs/"
+  private val compareDirString = "v2f/src/it/test-files/outputs-to-compare"
+
+  // the directory structure and files are created by the first test, so all we need to do is delete them afterwards
+  override def afterAll(): Unit = {
+    val file = File.apply(compareDirString)
+    file.delete()
+    ()
+  }
+
+  it should "write test data without throwing an error" in {
+    runWithRealContext(PipelineOptionsFactory.create()) { sc =>
+      ExtractionPipeline.convertAndWrite(sc, inputDirString, compareDirString)
+    }.waitUntilDone()
+  }
+
+  /**
+    *
+    * Helper method to parse the output files into a comparable format.
+    *
+    * @param directory The path to the directory where the files live.
+    * @param filePattern The glob pattern of files to read.
+    * @return One Set of Json that has every json object written to the output files.
+    */
+  private def createSetFromFiles(directory: File, filePattern: String): Set[Json] = {
+    directory
+      .glob(filePattern)
+      .flatMap { _.lineIterator }
```

Comment:
Slightly more idiomatic style:
```suggestion
      .flatMap(_.lineIterator)
```